From d30812f7afc355269df0edd5d4f030d470192cad Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Wed, 17 Aug 2016 13:28:32 +0500
Subject: [PATCH 2/2] install the vulkan loader

The vulkan loader is an essential component so it should
be installed to the directory that is intended to be
pushed to the target.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 loader/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/loader/CMakeLists.txt b/loader/CMakeLists.txt
index a4d2b21..2790faa 100644
--- a/loader/CMakeLists.txt
+++ b/loader/CMakeLists.txt
@@ -84,4 +84,5 @@ else()
     add_library(vulkan SHARED ${LOADER_SRCS})
     set_target_properties(vulkan PROPERTIES SOVERSION "1" VERSION "1.0.26")
     target_link_libraries(vulkan -ldl -lpthread -lm)
+    install(TARGETS vulkan DESTINATION ${CMAKE_INSTALL_LIBDIR})
 endif()
-- 
1.9.1

